content/docs/dvclive/get-started.md

```diff
@@ -1,134 +1,129 @@
 
-</admon>
-
-## Workflow
-
-### Initialize DVCLive
+In this example we are using [Keras](/doc/dvclive/ml-frameworks/keras) for
+training our model and we can add DVCLive as follows:
 
 ```python
-from dvclive import Live
-
-live = Live()
+# train.py
+from dvclive.keras import DvcLiveCallback
+...
+model.fit(
+  train_dataset, validation_data=validation_dataset,
+  callbacks=[DvcLiveCallback(path="training_metrics")])
 ```
 
-See [`Live()`](/doc/dvclive/api-reference/live) for details.
-
-### Log data
+You can find example snippets and reference of the options available in the
+corresponding page of any other of the supported
+[ML Frameworks](/doc/dvclive/ml-frameworks).
 
-<toggle>
-<tab title="Scalars">
+If you prefer to use the DVCLive API directly, check the
+[API Reference](/doc/dvclive/api-reference).
 
-```python
-live.log("acc", 0.9)
-```
+## Set up DVC
 
-See `Live.log()`.
+In order to visualize the outputs generated by DVCLive, we can set up a DVC
+pipeline.
 
-</tab>
-<tab title="Images">
+First, we initialize DVC with `dvc init`:
 
-```python
-img = np.ones((500, 500, 3), np.uint8)
-live.log_image("image.png", img)
+```dvc
+$ dvc init
 ```
 
-See `Live.log_image()`.
+After that, we use `dvc exp init --live` to create a `dvc.yaml` file:
 
-</tab>
-<tab title="Plots">
-
-```python
-y_true = [0, 0, 1, 1]
-y_pred = [0.2, 0.5, 0.3, 0.8]
-live.log_plot("roc", y_true, y_score)
-```
-
-See `Live.log_plot()`.
+<admon type="info">
 
-</tab>
-</toggle>
+Note that the path passed to the `--live` option (`"training_metrics"`) matches
+the `path` passed to the callback in the Python code.
 
-### (Optionally) Update the step number
+</admon>
 
-```python
-live.next_step()
+```dvc
+$ dvc exp init \
+--live "training_metrics" \
+--code "train.py" \
+python train.py
 ```
 
-See `Live.next_step()` and `Live.set_step()` for details.
-
-## Putting all together
-
-Joining the above snippets, you can include DVCLive in your training code:
-
-```python
-# train.py
-
-from dvclive import Live
-
-live = Live()
+<details>
+
+### ⚙️ Expand to see the `dvc.yaml`
+
+The `dvc.yaml` will contain a new `train` stage using the DVCLive outputs as
+`metrics` and `plots`:
+
+```yaml
+stages:
+  train:
+    cmd: python train.py
+    deps:
+      - train.py
+    metrics:
+      - training_metrics.json:
+          cache: false
+    plots:
+      - training_metrics/scalars:
+          cache: false
+```
 
-for epoch in range(NUM_EPOCHS):
-    train_model(...)
-    metrics = evaluate_model(...)
+You can find more details about the structure of files generated by DVCLive in
+the [API Reference](/doc/dvclive/api-reference#outputs)
 
-    for metric_name, value in metrics.items():
-        live.log(metric_name, value)
+</details>
 
-    live.next_step()
-```
+## Run experiments
 
-## Outputs
-
-After you run your training code, you should see the following content in the
-project:
+You can now
+[run experiments](/doc/user-guide/experiment-management/running-experiments)
```

Comment:
Maybe this should be more generic like "Run your code" and mention the non-DVC workflow as well, even if the examples continue with `dvc exp run`.

Reply:
Also, do we want to cover plain DVC integration (non-experiments i.e. persistent outputs + `dvc repro`) somewhere?

Reply:
I don't think we need to include `dvc repro` since users will get pretty much the same value out of `dvc exp run` here.